app/src/pages/inside/projectSettingsPageContainer/content/elements/notificationRuleContent/notificationRuleContent.jsx

```diff
@@ -19,6 +19,7 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import { defineMessages, useIntl } from 'react-intl';
 import { AttributeListContainer } from 'components/containers/attributeListContainer';
+import { ATTRIBUTES_OPERATORS } from 'pages/common/settingsPage/notificationsTab/constants';
```

Comment:
Please use corresponding constants from the redesigned components (import from '../../notifications/constants' as below).